BUILDING.md

```diff
@@ -155,3 +155,30 @@ installed to be able to compile the C/C++ sources.
 > cmake .. -GNinja -DCMAKE_BUILD_TYPE=Release
 > cmake --build .
 ```
+
+## Building with make (on UNIX systems)
+
+NOTE: `make` used to be gRPC's default build system, but we're no longer recommending it. You should use `bazel` or `cmake` instead.
+While still supported, make should only be used inside the gRPC repository (to satisfy some project-internal needs)
```

Comment:
"inside the gRPC repository" is maybe a little ambiguous. Perhaps "The makefile is only intended for internal usage and is not meant for public consumption.".